contracts/vesting/src/vesting/account/mixnode_bonding_account.rs

```diff
@@ -6,12 +6,34 @@ use cosmwasm_std::{wasm_execute, Coin, Env, Response, Storage, Uint128};
 use mixnet_contract_common::{ExecuteMsg as MixnetExecuteMsg, MixNode};
 use vesting_contract_common::events::{
     new_vesting_mixnode_bonding_event, new_vesting_mixnode_unbonding_event,
+    new_vesting_update_mixnode_config_event,
 };
 use vesting_contract_common::one_unym;
```

Comment:
Not necessarily part of this PR, but this function name also shouldn't make reference to token name

Reply:
These are both good calls, I'll fix it